### DISCUSSION TITLE
Duda github

### DISCUSSION THREAD

**Discussion by LeonardoSoriano** [Launch-X-Latam/MisionFrontEnd#229] [General]
Disculpen una duda, la practica de HTML, subimos el código en un readme o hay otra forma de subir el código?

**Comment by doguedogue** [comment]
en un readme pones la ruta y la puedes subir en github pages para que se vea desplegada

**Comment by juanmadaz** [comment]
Puedes subir los archivos directamente de tu PC en el botón Add file/Upload files, te servirá para tener un respaldo de tus archivos también.
